### DISCUSSION TITLE
Cannot update to 4.9.0-0.okd-2021-12-12-025847

### DISCUSSION THREAD

**Discussion by cronchiq** [okd-project/okd#1070] [General]
Cannot upgrade OKD cluster from 4.8.0-0.okd-2021-11-14-052418 to 4.9.0-0.okd-2021-12-12-025847. Progress is stuck at 91%.

**ClusterID**: 008bfbb1-dbbf-4236-b1a2-dfbaf0b66f11
**ClusterVersion**: Updating to "4.9.0-0.okd-2021-12-12-025847" from "4.9.0-0.okd-2021-11-28-035710" for 41 hours: Unable to apply 4.9.0-0.okd-2021-12-12-025847: wait has exceeded 40 minutes for these operators: authentication
**ClusterOperators**:
        clusteroperator/authentication is degraded because CustomRouteControllerDegraded: Ingress.config.openshift.io "cluster" is invalid: [status.componentRoutes.currentHostnames: Invalid value: "oauth-openshift.apps.njzmp.okd.cz.o2": status.componentRoutes.currentHostnames in body must be of type hostname: "oauth-openshift.apps.njzmp.okd.cz.o2", status.componentRoutes.defaultHostname: Invalid value: "oauth-openshift.apps.njzmp.okd.cz.o2": status.componentRoutes.defaultHostname in body must be of type hostname: "oauth-openshift.apps.njzmp.okd.cz.o2"]

**must-gather**:  https://drive.google.com/file/d/1ApTbYcm3F6wTcDdCb03v8TDJU69Zzkwz/view?usp=sharing

**Comment by vrutkovs** [comment]
see https://bugzilla.redhat.com/show_bug.cgi?id=2039256

**Reply by cronchiq** [reply]
we have been using this domain since the beginning. So what has changed?

Is there any WA to fix it?
